.jenkins

```diff
@@ -135,82 +135,86 @@ pipeline {
                         }
                     }
                 }
-//                stage('HIP-ROCm-5.2') {
-//                    agent {
-//                        dockerfile {
-//                            filename 'Dockerfile.hipcc'
-//                            dir 'scripts/docker'
-//                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-20.04:5.2'
-//                            label 'rocm-docker && vega'
-//                            args '-v /tmp/ccache.kokkos:/tmp/ccache --device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video --env HIP_VISIBLE_DEVICES=$HIP_VISIBLE_DEVICES'
-//                        }
-//                    }
-//                    environment {
-//                        OMP_NUM_THREADS = 8
-//                        OMP_MAX_ACTIVE_LEVELS = 3
-//                        OMP_PLACES = 'threads'
-//                        OMP_PROC_BIND = 'spread'
-//                    }
-//                    steps {
-//                        sh 'ccache --zero-stats'
-//                        sh 'echo "/opt/rocm/llvm/lib" > /etc/ld.so.conf.d/llvm.conf && ldconfig'
-//                        sh '''rm -rf build && mkdir -p build && cd build && \
-//                              cmake \
-//                                -DCMAKE_BUILD_TYPE=Debug \
-//                                -DCMAKE_CXX_COMPILER=hipcc \
-//                                -DCMAKE_CXX_FLAGS="-Werror -Wno-unused-command-line-argument -DNDEBUG" \
-//                                -DCMAKE_CXX_STANDARD=17 \
-//                                -DKokkos_ARCH_NATIVE=ON \
-//                                -DKokkos_ENABLE_COMPILER_WARNINGS=ON \
-//                                -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
-//                                -DKokkos_ENABLE_TESTS=ON \
-//                                -DKokkos_ENABLE_BENCHMARKS=ON \
-//                                -DKokkos_ENABLE_HIP=ON \
-//                                -DKokkos_ENABLE_OPENMP=ON \
-//                                -DKokkos_ENABLE_HIP_MULTIPLE_KERNEL_INSTANTIATIONS=ON \
-//                              .. && \
+                stage('HIP-ROCm-5.2') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.hipcc'
+                            dir 'scripts/docker'
+                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-20.04:5.2'
+                            label 'nvidia-docker && ampere'
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache --device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video --env HIP_VISIBLE_DEVICES=$HIP_VISIBLE_DEVICES'
```

Comment:
No error with the bogus args?

Reply:
Did it give an error?

Reply:
The previous try didn't work but there was no error message. Might be that or something else, I don't know.